src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+public class Deadline extends Task{
+    protected String by;
+    protected static String TYPE = "deadline";
+    public Deadline(String description, String deadLine) {
+        super(description);
+        this.by = deadLine;
+    }
+
```

Comment:
Consider adding @Override for clarity that this is an overridden function from its superclass.